README.md

```diff
@@ -77,7 +69,8 @@ See also environment variables from lower layers: [`docker-openjdk`](https://git
 | JETTY_PORT_SSL | |  | 8443 | solr6 only |
 | GLOBAL_WORKSPACE_\<variable\> | \<variable\> | | | for workspace core or shards |
 | GLOBAL_ARCHIVE_\<variable\> | \<variable\> | | | for archive core |
-| GLOBAL_\<variable\> | \<variable> | | |  |
+| GLOBAL_ALL_\<variable\> | \<variable> | | | for all cores |
+| GLOBAL_<core>_\<variable\> | \<variable> | | | for specific core |
```

Comment:
With sharding, are there more other cores being used, besides `workspace` and `archive`, so that you had to add a new group variables here?

Reply:
Not sure you understood what I am doing here:

The <core> in this context is the shard-core, e.g. alfresco-0, alfresco-1

Reply:
Aha, okay. I was not aware of how shards work